tools/cldr/gendatefmts2.js

```diff
@@ -43,14 +43,137 @@ var hardCodeData = {
         "ordinalChoice": "#{num}天"
     },
     "am": {
-      "a0-ethiopic": "ጥዋት",
-      "a1-ethiopic": "ቀትር",
-      "a2-ethiopic": "ከሰዓት",
-      "a3-ethiopic": "ከምሽቱ",
-      "a4-ethiopic": "ከሌሊቱ",
+        "a0-ethiopic": "ጥዋት",
+        "a1-ethiopic": "ቀትር",
+        "a2-ethiopic": "ከሰዓት",
+        "a3-ethiopic": "ከምሽቱ",
+        "a4-ethiopic": "ከሌሊቱ",
     },
     "de": {
         "ordinalChoice": "#{num}."
+    },
+    "my": {
+        "1#1 sec|#{num} sec": "#{num} စက္ကန့်"
+    },
+    "tg": {
+        "1#1 millisecond|#{num} milliseconds": "#{num} миллисекундҳо",
+        "1#1 second|#{num} seconds": "#{num} сония",
+        "1#1 minute|#{num} minutes": "#{num} дақиқа",
+        "1#1 hour|#{num} hours": "#{num} соат",
+        "1#1 day|#{num} days": "#{num} рӯз",
+        "1#1 week|#{num} weeks": "#{num} ҳафта",
+        "1#1 month|#{num} months": "#{num} моҳ",
+        "1#1 year|#{num} years": "#{num} сол",
+        "#{num} ms": "#{num} мил",
+        "1#1 sec|#{num} sec": "#{num} сон",
+        "1#1 min|#{num} min": "#{num} дақ",
+        "1#1 hr|#{num} hrs": "#{num} соа",
+        "durationLongDays": "#{num} рӯз",
+        "1#1 wk|#{num} wks": "#{num} ҳаф",
+        "1#1 mon|#{num} mons": "#{num} моҳ",
+        "1#1 yr|#{num} yrs": "#{num} сол",
+        "durationMediumMillis": "#{num} ми",
+        "1#1 se|#{num} sec": "#{num} со",
+        "1#1 mi|#{num} min": "#{num} да",
+        "durationMediumHours": "#{num} со",
+        "1#1 dy|#{num} dys": "#{num} рӯ",
+        "durationMediumWeeks": "#{num} ҳа",
+        "1#1 mo|#{num} mos": "#{num} мо",
+        "durationMediumYears": "#{num} со",
+        "durationShortMillis": "#{num} м",
+        "#{num}s": "#{num} с",
+        "durationShortMinutes": "#{num} д",
+        "#{num}h": "#{num} с",
+        "#{num}d": "#{num} р",
+        "#{num}w": "#{num} ҳ",
+        "durationShortMonths": "#{num} м",
+        "#{num}y": "#{num} с"
+    },
+    "mt": {
+        "1#1 second|#{num} seconds": "one#{num} sekonda|few#{num} sekondi|many#{num}-il sekonda|#{num} sekonda",
```

Comment:
One of  concern is if we add these parts as hardcode, We may miss the chance to get next cldr updates

Reply:
That is true... we will have to be vigilant when we update to CLDR41 to see if CLDR has added those locales yet.